app/assets/javascripts/discourse/app/controllers/preferences.js

```diff
@@ -1,2 +1,12 @@
+import { and, readOnly } from "@ember/object/computed";
 import Controller from "@ember/controller";
-export default Controller.extend({});
+
+export default Controller.extend({
+  canChangeCategoryPreferences: readOnly(
+    "model.can_change_tracking_preferences"
+  ),
+  canChangeTagPreferences: and(
```

Comment:
Minor but if you are only using this in a template you can use the `(and` helper now which will eliminate some code.

Reply:
I was aware that was possible, but couldn't find an example of it. TIL it's `(and`.